src/sql/workbench/services/connection/browser/connectionWidget.ts

```diff
@@ -40,7 +40,8 @@ export enum AuthenticationType {
 	SqlLogin = 'SqlLogin',
 	Integrated = 'Integrated',
 	AzureMFA = 'AzureMFA',
-	AzureMFAAndUser = 'AzureMFAAndUser'
+	AzureMFAAndUser = 'AzureMFAAndUser',
+	dSTSAuth = 'dstsAuth'
```

Comment:
it's not really following naming convention to start a constant with a lower-case character.  I guess `DSTSAuth` or `DstsAuth` aren't really improvements here?